omod-2.2/src/test/resources/obsDataSet2_2.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<dataset>
```

Comment:
Why not edit an existing obs and avoid adding yet another test dataset file?

Reply:
Oh thats a good idea

Reply:
When I try to use the other dataset [obsWithGroupMembers](https://github.com/openmrs/openmrs-module-webservices.rest/blob/master/omod-1.8/src/test/resources/obsWithGroupMembers.xml) shows 
>**org.dbunit.DatabaseUnitRuntimeException: org.dbunit.DatabaseUnitException: Exception processing table name='obs'** error

Reply:
You do not have to load any dataset. Just use any value in this file: https://github.com/openmrs/openmrs-core/blob/master/api/src/test/resources/org/openmrs/include/standardTestDataset.xml

Reply:
But @dkayiwa this also is not accessible 
>org.dbunit.DatabaseUnitRuntimeException: java.io.FileNotFoundException: Unable to find 'standardTestDataset.xml' in the classpath

Reply:
You do not need to access it. It should already be part of the available test data, load by some base class.

Reply:
Am failing to use it ...when I try since I cannot execute it .....like not to execute any test dataset but to get the uuid it is failing again

Reply:
You do not need to execute it. It is automatically executed for you by a base class. You just need to use a uuid from it.